livekit-plugins/livekit-plugins-llama-index/README.md

```diff
@@ -1,14 +1,36 @@
-# LiveKit Plugins Minimal
+# LiveKit Plugins Llama Index
 
-This is a minimal example of a LiveKit plugin for Agents.
+Agent Framework plugin for using Llama Index. Currently supports [Query Engine](https://docs.llamaindex.ai/en/stable/module_guides/deploying/query_engine/) and [Chat Engine](https://docs.llamaindex.ai/en/stable/module_guides/deploying/chat_engines/).
 
-### Developer note
+## Install
 
-When copying this directory over to create a new `livekit-plugins` package, make sure it's nested within the `livekit-plugins` folder and that the `"name"` field in `package.json` follows the proper naming convention for CI:
+```bash
+pip install livekit-plugins-llama-index
+```
+
+## Query Engine
+
+Query Engine is primarily used for RAG. See [example voice agent](https://github.com/livekit/agents/blob/main/examples/voice-pipeline-agent/llamaindex-rag/query_engine.py)
+
+## Chat Engine
 
-```json
-{
-  "name": "livekit-plugins-<name>",
-  "private": true
-}
+Chat Engine can be used as an LLM within the frame work.
```

Comment:
ChatEngine isn't the recommended way tho. Since this one doesn't support function calls (It is using the LLM from lama_index)

Reply:
This one: https://github.com/livekit/agents/blob/main/examples/voice-pipeline-agent/llamaindex-rag/query_engine.py
and this one https://github.com/livekit/agents/blob/main/examples/voice-pipeline-agent/llamaindex-rag/retrieval.py
are more appropriate, but it depends on the needs.

The ChatEngine is also doing some "context" management (like keeping the previous rag search of the prior question etc.). They all have pros and cons

Reply:
I see, you have an example for it, I'm just documenting it.